modules/social_features/social_content_report/src/ContentTranslationDefaultsConfigOverride.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace Drupal\social_content_report;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorableConfigBase;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+
+/**
+ * Provides content translation for the Social Content Report module.
+ *
+ * @package Drupal\social_content_report
+ */
+class ContentTranslationDefaultsConfigOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The module handler.
+   *
+   * @var \Drupal\Core\Extension\ModuleHandlerInterface
+   */
+  protected $moduleHandler;
+
+  /**
+   * ContentTranslationDefaultsConfigOverride constructor.
+   *
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   The module handler service.
+   */
+  public function __construct(ModuleHandlerInterface $module_handler) {
+    $this->moduleHandler = $module_handler;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function loadOverrides($names): array {
```

Comment:
If possible, let's try our best not to introduce another config override, we are trying to move away from these overrides and get rid of them in long term.

Reply:
Hi @navneet0693 !
Unfortunately, if we want to avoid using config overrides we need to do this globally and replace all config overrides for translations (and this will take time). So, I suggest leaving this for now as it is and maybe, create a follow-up for replacing config overrides globally for translations.